draft-ietf-quic-recovery.md

```diff
@@ -1153,8 +1174,9 @@ Pseudocode for OnLossDetectionTimeout follows:
 
 ~~~
 OnLossDetectionTimeout():
-  loss_time, pn_space = GetEarliestLossTime()
-  if (loss_time != 0):
+  earliest_loss_time, pn_space =
+    GetEarliestTimeAndSpace(loss_times)
```

Comment:
This should be `loss_time` instead of `loss_times`. `loss_time` is a global variable.

Reply:
Thanks, I missed that one.